src/components/gcds-site-menu/gcds-site-menu.css

```diff
@@ -0,0 +1,981 @@
+.h2-mobile-menu-body-lock {
+  overflow: hidden;
+}
+@media screen and (min-width: 64em) {
+  .h2-mobile-menu-body-lock {
+    overflow: visible;
+  }
+}
+
+:host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger] {
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 10000000;
+}
+@media screen and (min-width: 64em) {
+  :host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger] {
+    display: none;
+  }
+}
+:host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger] [data-h2-mobile-menu-trigger-open-label] {
+  display: inline;
+}
+:host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger] [data-h2-mobile-menu-trigger-close-label] {
+  display: none;
+}
+:host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger].h2-active [data-h2-mobile-menu-trigger-open-label] {
+  display: none;
+}
+:host([data-h2-menu-wrapper]) [data-h2-mobile-menu-trigger].h2-active [data-h2-mobile-menu-trigger-close-label] {
+  display: inline;
+}
+:host([menu-mobile-layout*=drawer]) [data-h2-menu] {
```

Comment:
Can we combine the styling for `menu-mobile-layout*=drawer` and `menu-mobile-layout*=toolbar` whenever they have styling in common? For example in this case:
```
:host([menu-mobile-layout*=drawer]) [data-h2-menu],
:host([menu-mobile-layout*=toolbar]) [data-h2-menu] {
  position: fixed;
  left: 0;
  width: 100vw;
  z-index: 1000000;
}
```